gammapy/data/observations.py

```diff
@@ -124,25 +127,34 @@ def gti(self):
         return gti
 
     @staticmethod
-    def _get_obs_info(pointing, deadtime_fraction):
+    def _get_obs_info(pointing, deadtime_fraction, time_start, time_stop, reference_time, location):
         """Create obs info dict from in memory data"""
-        return {
+        obs_info = {
             "RA_PNT": pointing.icrs.ra.deg,
             "DEC_PNT": pointing.icrs.dec.deg,
             "DEADC": 1 - deadtime_fraction,
         }
+        obs_info.update(time_ref_to_dict(reference_time))
+        obs_info['TSTART'] = time_relative_to_ref(time_start, obs_info).to_value(u.s)
+        obs_info['TSTOP'] = time_relative_to_ref(time_stop, obs_info).to_value(u.s)
```

Comment:
If `obs_info` contains only relative time we have to also store the reference time information, no?

Also it might be simpler to pass the GTI directly.

Reply:
The reference time is stored above in line 137:

`obs_info.update(time_ref_to_dict(reference_time))`

Reply:
My bad. Thanks.